rpm/iptables.spec

```diff
@@ -104,6 +105,9 @@ install -c -m 755 ip6tables-config %{buildroot}/etc/sysconfig/ip6tables-config
 %{_libdir}/xtables/libxt*
 %{_libdir}/libip*tc.so.*
 %{_libdir}/libxtables.so.*
+# Exclude nftables tools built with 1.8.11 even with ntfables being disabled
+%exclude /sbin/nfnl_osf
+%exclude %{_datadir}/xtables/pf.os
 
 %files devel
 %defattr(-,root,root,-)
```

Comment:
Remove the not needed "%defattr(-,root,root,-)" from all files sections.

Reply:
Dropped in a192cfc6000334aa643f0db764fead6dff5e7bbc